tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/RunCspellTool/RunCspellTool.cs

```diff
@@ -0,0 +1,290 @@
+            Command command = new("cspell", "Run cspell to check for typos in the specified package");
+            command.AddArgument(_packagePathArg);
+            command.SetHandler(async ctx => { await HandleCommand(ctx, ctx.GetCancellationToken()); });
+            return command;
+        }
+
+        public override async Task HandleCommand(InvocationContext ctx, CancellationToken ct)
+        {
+            string packagePath = ctx.ParseResult.GetValueForArgument(_packagePathArg);
+            var result = await RunCspellCheck(packagePath);
+            ctx.ExitCode = ExitCode;
+            output.Output(result);
+        }
+
+        [McpServerTool(Name = "run-cspell"), Description("Run cspell to check for typos in the specified package path")]
+        public async Task<DefaultCommandResponse> RunCspellCheck(string packagePath)
+        {
+            try
+            {
+                logger.LogInformation("Running cspell check for package path: {packagePath}", packagePath);
+
+                if (string.IsNullOrWhiteSpace(packagePath))
+                {
+                    SetFailure(1);
+                    return new DefaultCommandResponse
+                    {
+                        ResponseError = "Package path is required"
+                    };
+                }
+
+                if (!Directory.Exists(packagePath))
+                {
+                    SetFailure(1);
+                    return new DefaultCommandResponse
+                    {
+                        ResponseError = $"Package path does not exist: {packagePath}"
+                    };
+                }
+
+                // Get the absolute path to the Invoke-Cspell.ps1 script
+                var scriptPath = GetInvokeCspellScriptPath();
+                if (string.IsNullOrEmpty(scriptPath))
+                {
+                    SetFailure(1);
+                    return new DefaultCommandResponse
+                    {
+                        ResponseError = "Could not locate Invoke-Cspell.ps1 script"
+                    };
+                }
+
+                var startTime = DateTime.UtcNow;
+                var output = await RunPowerShellScript(scriptPath, packagePath);
+                var duration = (DateTime.UtcNow - startTime).TotalMilliseconds;
+
+                // Parse the cspell output to determine success/failure and extract typos
+                var result = ParseCspellOutput(output);
+
+                return new DefaultCommandResponse
+                {
+                    Message = result.Success ? "Cspell check completed successfully" : "Cspell check found spelling errors",
+                    Result = new
+                    {
+                        Success = result.Success,
+                        TyposFound = result.TyposFound,
+                        Output = result.Output,
+                        Guidance = GetGuidanceMessage(result)
+                    },
+                    Duration = (long)duration
```

Comment:
```suggestion
```

I don't think we really need this and I'm probably going to remove it. Unless `duration` is something that matters in a cspell flow?